platform/ios/MGLMapboxEvents.m

```diff
@@ -136,7 +195,26 @@ - (instancetype) init {
             
             [[NSUserDefaults standardUserDefaults] registerDefaults:defaultsToRegister];
         }
+    }
+}
+
++ (BOOL)isEnabled {
+    return ([[NSUserDefaults standardUserDefaults] boolForKey:@"MGLMapboxMetricsEnabled"] &&
+            [[NSUserDefaults standardUserDefaults] integerForKey:@"MGLMapboxAccountType"] == 0);
```

Comment:
Nah, I think we're ok then. We have a tenuous expression of that dependency here, but I think it's fair.